Fail to build current main on Mac

I'm trying to build the current main on my Mac (so that I can run the tests locally for development). However, I fail to build the included `region` module with
```
regparser.y:629:5: error: implicit declaration of function 'regYYrestart' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
    regYYrestart(NULL);
    ^
1 error generated.
```
(I paste more of the relevant output at the bottom). 

I have a feeling that I've seen this before and there is just some simple complier flag that I need to set, but I don't find it anywhere (maybe it's in confluence that I don't have access to or it was in an old email I don't find any longer?)
@Marie-Terrell Maybe you know? Please don't spend a lot of time debugging, but I thought I'll quickly ask, just in case.

This is with Apple's clang from XCode
```
Apple clang version 12.0.0 (clang-1200.0.32.29)
Target: x86_64-apple-darwin19.6.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```


Relevant output below. I can provide the full screen output, but I want to save you the time to scroll through all the pringin for the modules that do build.
```
=== configuring in region-4.13 (/Users/hamogu/code/sherpa/extern/region-4.13)
configure: running /bin/sh ./configure --disable-option-checking '--prefix=/Users/hamogu/code/sherpa/build'  '--with-pic' '--enable-standalone' '--disable-maintainer-mode' '--enable-stuberrorlib' '--disable-shared' '--enable-shared=libgrp,stklib' '--enable-fftw' '--enable-region' '--enable-group' '--enable-stk' '--enable-wcs' --cache-file=/dev/null --srcdir=.
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... ./install-sh -c -d
checking for gawk... no
checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether to enable maintainer-specific portions of Makefiles... no
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of gcc... gcc3
checking for bison... bison -y
checking for flex... flex
checking lex output file root... lex.yy
checking lex library... -ll
checking whether yytext is a pointer... yes
checking for a sed that does not truncate output... /usr/bin/sed
checking build system type... x86_64-apple-darwin19.6.0
checking host system type... x86_64-apple-darwin19.6.0
checking how to print strings... printf
checking for a sed that does not truncate output... (cached) /usr/bin/sed
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for fgrep... /usr/bin/grep -F
checking for ld used by gcc... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld
checking if the linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) is GNU ld... no
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 196608
checking how to convert x86_64-apple-darwin19.6.0 file names to x86_64-apple-darwin19.6.0 format... func_convert_file_noop
checking how to convert x86_64-apple-darwin19.6.0 file names to toolchain format... func_convert_file_noop
checking for /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... no
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from gcc object... ok
checking for sysroot... no
checking for a working dd... /bin/dd
checking how to truncate binary pipes... /bin/dd bs=4096 count=1
checking for mt... no
checking if : is a manifest tool... no
checking for dsymutil... dsymutil
checking for nmedit... nmedit
checking for lipo... lipo
checking for otool... otool
checking for otool64... no
checking for -single_module linker flag... yes
checking for -exported_symbols_list linker flag... yes
checking for -force_load linker flag... yes
checking how to run the C preprocessor... gcc -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... yes
checking for gcc option to produce PIC... -fno-common -DPIC
checking if gcc PIC flag -fno-common -DPIC works... yes
checking if gcc static flag -static works... no
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) supports shared libraries... yes
checking dynamic linker characteristics... darwin19.6.0 dyld
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... no
checking whether to build static libraries... yes
checking for main in -lm... yes
checking for main in -lregion... no
checking float.h usability... yes
checking float.h presence... yes
checking for float.h... yes
checking for stdlib.h... (cached) yes
checking for string.h... (cached) yes
checking for stdbool.h that conforms to C99... yes
checking for _Bool... yes
checking for stdlib.h... (cached) yes
checking for GNU libc compatible malloc... yes
checking for stdlib.h... (cached) yes
checking for GNU libc compatible realloc... yes
checking for sqrt... yes
checking for strcasecmp... yes
checking for strchr... yes
checking for strpbrk... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating src/Makefile
config.status: executing depfiles commands
config.status: executing libtool commands


... some lines removed ...


Making install in region-4.13
Making install in src
/Applications/Xcode.app/Contents/Developer/usr/bin/make  install-am
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DPACKAGE_NAME=\"region\" -DPACKAGE_TARNAME=\"region\" -DPACKAGE_VERSION=\"4.13\" -DPACKAGE_STRING=\"region\ 4.13\" -DPACKAGE_BUGREPORT=\"cxchelp@head.cfa.harvard.edu\" -DPACKAGE_URL=\"\" -DPACKAGE=\"region\" -DVERSION=\"4.13\" -DYYTEXT_POINTER=1 -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_LIBM=1 -DHAVE_FLOAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE__BOOL=1 -DHAVE_STDBOOL_H=1 -DHAVE_STDLIB_H=1 -DHAVE_MALLOC=1 -DHAVE_STDLIB_H=1 -DHAVE_REALLOC=1 -DHAVE_SQRT=1 -DHAVE_STRCASECMP=1 -DHAVE_STRCHR=1 -DHAVE_STRPBRK=1 -I.     -g -O2 -MT reglexer.lo -MD -MP -MF .deps/reglexer.Tpo -c -o reglexer.lo reglexer.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DPACKAGE_NAME=\"region\" -DPACKAGE_TARNAME=\"region\" -DPACKAGE_VERSION=\"4.13\" -DPACKAGE_STRING=\"region\ 4.13\" -DPACKAGE_BUGREPORT=\"cxchelp@head.cfa.harvard.edu\" -DPACKAGE_URL=\"\" -DPACKAGE=\"region\" -DVERSION=\"4.13\" -DYYTEXT_POINTER=1 -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_LIBM=1 -DHAVE_FLOAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE__BOOL=1 -DHAVE_STDBOOL_H=1 -DHAVE_STDLIB_H=1 -DHAVE_MALLOC=1 -DHAVE_STDLIB_H=1 -DHAVE_REALLOC=1 -DHAVE_SQRT=1 -DHAVE_STRCASECMP=1 -DHAVE_STRCHR=1 -DHAVE_STRPBRK=1 -I.     -g -O2 -MT regparser.lo -MD -MP -MF .deps/regparser.Tpo -c -o regparser.lo regparser.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DPACKAGE_NAME=\"region\" -DPACKAGE_TARNAME=\"region\" -DPACKAGE_VERSION=\"4.13\" -DPACKAGE_STRING=\"region\ 4.13\" -DPACKAGE_BUGREPORT=\"cxchelp@head.cfa.harvard.edu\" -DPACKAGE_URL=\"\" -DPACKAGE=\"region\" -DVERSION=\"4.13\" -DYYTEXT_POINTER=1 -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_LIBM=1 -DHAVE_FLOAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE__BOOL=1 -DHAVE_STDBOOL_H=1 -DHAVE_STDLIB_H=1 -DHAVE_MALLOC=1 -DHAVE_STDLIB_H=1 -DHAVE_REALLOC=1 -DHAVE_SQRT=1 -DHAVE_STRCASECMP=1 -DHAVE_STRCHR=1 -DHAVE_STRPBRK=1 -I.     -g -O2 -MT region_area.lo -MD -MP -MF .deps/region_area.Tpo -c -o region_area.lo region_area.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DPACKAGE_NAME=\"region\" -DPACKAGE_TARNAME=\"region\" -DPACKAGE_VERSION=\"4.13\" -DPACKAGE_STRING=\"region\ 4.13\" -DPACKAGE_BUGREPORT=\"cxchelp@head.cfa.harvard.edu\" -DPACKAGE_URL=\"\" -DPACKAGE=\"region\" -DVERSION=\"4.13\" -DYYTEXT_POINTER=1 -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_LIBM=1 -DHAVE_FLOAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE__BOOL=1 -DHAVE_STDBOOL_H=1 -DHAVE_STDLIB_H=1 -DHAVE_MALLOC=1 -DHAVE_STDLIB_H=1 -DHAVE_REALLOC=1 -DHAVE_SQRT=1 -DHAVE_STRCASECMP=1 -DHAVE_STRCHR=1 -DHAVE_STRPBRK=1 -I.     -g -O2 -MT region_convert.lo -MD -MP -MF .deps/region_convert.Tpo -c -o region_convert.lo region_convert.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DPACKAGE_NAME=\"region\" -DPACKAGE_TARNAME=\"region\" -DPACKAGE_VERSION=\"4.13\" -DPACKAGE_STRING=\"region\ 4.13\" -DPACKAGE_BUGREPORT=\"cxchelp@head.cfa.harvard.edu\" -DPACKAGE_URL=\"\" -DPACKAGE=\"region\" -DVERSION=\"4.13\" -DYYTEXT_POINTER=1 -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_LIBM=1 -DHAVE_FLOAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE__BOOL=1 -DHAVE_STDBOOL_H=1 -DHAVE_STDLIB_H=1 -DHAVE_MALLOC=1 -DHAVE_STDLIB_H=1 -DHAVE_REALLOC=1 -DHAVE_SQRT=1 -DHAVE_STRCASECMP=1 -DHAVE_STRCHR=1 -DHAVE_STRPBRK=1 -I.     -g -O2 -MT region_create.lo -MD -MP -MF .deps/region_create.Tpo -c -o region_create.lo region_create.c
libtool: compile:  gcc -DPACKAGE_NAME=\"region\" -DPACKAGE_TARNAME=\"region\" -DPACKAGE_VERSION=\"4.13\" "-DPACKAGE_STRING=\"region 4.13\"" -DPACKAGE_BUGREPORT=\"cxchelp@head.cfa.harvard.edu\" -DPACKAGE_URL=\"\" -DPACKAGE=\"region\" -DVERSION=\"4.13\" -DYYTEXT_POINTER=1 -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_LIBM=1 -DHAVE_FLOAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE__BOOL=1 -DHAVE_STDBOOL_H=1 -DHAVE_STDLIB_H=1 -DHAVE_MALLOC=1 -DHAVE_STDLIB_H=1 -DHAVE_REALLOC=1 -DHAVE_SQRT=1 -DHAVE_STRCASECMP=1 -DHAVE_STRCHR=1 -DHAVE_STRPBRK=1 -I. -g -O2 -MT reglexer.lo -MD -MP -MF .deps/reglexer.Tpo -c reglexer.c  -fno-common -DPIC -o reglexer.o
libtool: compile:  gcc -DPACKAGE_NAME=\"region\" -DPACKAGE_TARNAME=\"region\" -DPACKAGE_VERSION=\"4.13\" "-DPACKAGE_STRING=\"region 4.13\"" -DPACKAGE_BUGREPORT=\"cxchelp@head.cfa.harvard.edu\" -DPACKAGE_URL=\"\" -DPACKAGE=\"region\" -DVERSION=\"4.13\" -DYYTEXT_POINTER=1 -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_LIBM=1 -DHAVE_FLOAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE__BOOL=1 -DHAVE_STDBOOL_H=1 -DHAVE_STDLIB_H=1 -DHAVE_MALLOC=1 -DHAVE_STDLIB_H=1 -DHAVE_REALLOC=1 -DHAVE_SQRT=1 -DHAVE_STRCASECMP=1 -DHAVE_STRCHR=1 -DHAVE_STRPBRK=1 -I. -g -O2 -MT region_area.lo -MD -MP -MF .deps/region_area.Tpo -c region_area.c  -fno-common -DPIC -o region_area.o
libtool: compile:  gcc -DPACKAGE_NAME=\"region\" -DPACKAGE_TARNAME=\"region\" -DPACKAGE_VERSION=\"4.13\" "-DPACKAGE_STRING=\"region 4.13\"" -DPACKAGE_BUGREPORT=\"cxchelp@head.cfa.harvard.edu\" -DPACKAGE_URL=\"\" -DPACKAGE=\"region\" -DVERSION=\"4.13\" -DYYTEXT_POINTER=1 -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_LIBM=1 -DHAVE_FLOAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE__BOOL=1 -DHAVE_STDBOOL_H=1 -DHAVE_STDLIB_H=1 -DHAVE_MALLOC=1 -DHAVE_STDLIB_H=1 -DHAVE_REALLOC=1 -DHAVE_SQRT=1 -DHAVE_STRCASECMP=1 -DHAVE_STRCHR=1 -DHAVE_STRPBRK=1 -I. -g -O2 -MT regparser.lo -MD -MP -MF .deps/regparser.Tpo -c regparser.c  -fno-common -DPIC -o regparser.o
libtool: compile:  gcc -DPACKAGE_NAME=\"region\" -DPACKAGE_TARNAME=\"region\" -DPACKAGE_VERSION=\"4.13\" "-DPACKAGE_STRING=\"region 4.13\"" -DPACKAGE_BUGREPORT=\"cxchelp@head.cfa.harvard.edu\" -DPACKAGE_URL=\"\" -DPACKAGE=\"region\" -DVERSION=\"4.13\" -DYYTEXT_POINTER=1 -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_LIBM=1 -DHAVE_FLOAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE__BOOL=1 -DHAVE_STDBOOL_H=1 -DHAVE_STDLIB_H=1 -DHAVE_MALLOC=1 -DHAVE_STDLIB_H=1 -DHAVE_REALLOC=1 -DHAVE_SQRT=1 -DHAVE_STRCASECMP=1 -DHAVE_STRCHR=1 -DHAVE_STRPBRK=1 -I. -g -O2 -MT region_create.lo -MD -MP -MF .deps/region_create.Tpo -c region_create.c  -fno-common -DPIC -o region_create.o
libtool: compile:  gcc -DPACKAGE_NAME=\"region\" -DPACKAGE_TARNAME=\"region\" -DPACKAGE_VERSION=\"4.13\" "-DPACKAGE_STRING=\"region 4.13\"" -DPACKAGE_BUGREPORT=\"cxchelp@head.cfa.harvard.edu\" -DPACKAGE_URL=\"\" -DPACKAGE=\"region\" -DVERSION=\"4.13\" -DYYTEXT_POINTER=1 -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_LIBM=1 -DHAVE_FLOAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE__BOOL=1 -DHAVE_STDBOOL_H=1 -DHAVE_STDLIB_H=1 -DHAVE_MALLOC=1 -DHAVE_STDLIB_H=1 -DHAVE_REALLOC=1 -DHAVE_SQRT=1 -DHAVE_STRCASECMP=1 -DHAVE_STRCHR=1 -DHAVE_STRPBRK=1 -I. -g -O2 -MT region_convert.lo -MD -MP -MF .deps/region_convert.Tpo -c region_convert.c  -fno-common -DPIC -o region_convert.o
regparser.y:629:5: error: implicit declaration of function 'regYYrestart' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
    regYYrestart(NULL);
    ^
1 error generated.
mv -f .deps/region_create.Tpo .deps/region_create.Plo
make[3]: *** [regparser.lo] Error 1
make[3]: *** Waiting for unfinished jobs....
mv -f .deps/region_convert.Tpo .deps/region_convert.Plo
mv -f .deps/region_area.Tpo .deps/region_area.Plo
mv -f .deps/reglexer.Tpo .deps/reglexer.Plo
make[2]: *** [install] Error 2
make[1]: *** [install-recursive] Error 1
make: *** [install-recursive] Error 1
```

--- a/docs/install.rst
+++ b/docs/install.rst
@@ -390,7 +390,7 @@ be used when building and installing Sherpa.
    (Note that on MacOS, "gcc" is usally just an alias to Clang. Unless
    you specifically intall a gcc from a different source, you are
    likely to run into this problem, even if you believe that your
-   compiler is invoced with `gcc`.)
+   compiler is invoked with `gcc`.)
 
 
 A standard installation

--- a/docs/install.rst
+++ b/docs/install.rst
@@ -387,8 +387,8 @@ be used when building and installing Sherpa.
 
      CFLAGS='-Wno-implicit-function-declaration'
 
-   (Note that on MacOS, "gcc" is usally just an alias to Clang. Unless
-   you specifically intall a gcc from a different source, you are
+   (Note that on MacOS, "gcc" is usually just an alias to Clang. Unless
+   you specifically install a gcc from a different source, you are
    likely to run into this problem, even if you believe that your
    compiler is invoked with `gcc`.)
 

--- a/docs/install.rst
+++ b/docs/install.rst
@@ -84,7 +84,7 @@ Citing Sherpa
 -------------
 
 Information on citing Sherpa can be found from the
-`CITATION document <https://github.com/sherpa/sherpa/blob/master/CITATION>`_
+`CITATION document <https://github.com/sherpa/sherpa/blob/main/CITATION>`_
 in the Sherpa repository, or from the
 `Sherpa Zenodo page <https://doi.org/10.5281/zenodo.593753>`_.
 
@@ -167,7 +167,7 @@ or from
 either a release version,
 such as the
 `4.10.0 <https://github.com/sherpa/sherpa/tree/4.10.0>`_ tag,
-or the ``master`` branch (which is not guaranteed to be stable).
+or the ``main`` branch (which is not guaranteed to be stable).
 
 For example::
 
@@ -377,6 +377,21 @@ be used when building and installing Sherpa.
      export PYTHON_LDFLAGS=' '
 
    That is, the variable is set to a space, not the empty string.
+
+.. note::
+
+   Additionally, if you are building with Clang version 12, you will
+   encounter build issues in the region area related to an implicit
+   declaration. If so, pre-pending the following to your pip install
+   or python setup.py install line should resolve the build issues::
+
+     CFLAGS='-Wno-implicit-function-declaration'
+
+   (Note that on MacOS, "gcc" is usally just an alias to Clang. Unless
+   you specifically intall a gcc from a different source, you are
+   likely to run into this problem, even if you believe that your
+   compiler is invoced with `gcc`.)
+
 
 A standard installation
 ^^^^^^^^^^^^^^^^^^^^^^^

